src/test/regress/sql/schema_based_sharding.sql

```diff
@@ -65,6 +65,10 @@ SELECT undistribute_table('tenant_2.test_table');
 SELECT alter_distributed_table('tenant_2.test_table', colocate_with => 'none');
 -- verify we also don't allow colocate_with a tenant table
 SELECT alter_distributed_table('regular_schema.test_table', colocate_with => 'tenant_2.test_table');
+-- verify we don't allow ALTER TABLE SET SCHEMA for tenant tables
+ALTER TABLE tenant_2.test_table SET SCHEMA regular_schema;
+-- verify we don't allow ALTER TABLE SET SCHEMA for tenant schemas
+ALTER TABLE regular_schema.test_table SET SCHEMA tenant_2;
```

Comment:
maybe one more tests where try to alter schema of "tenant_1.users" to "tenant_2"

Reply:
and one more, where we try to alter schema of "regular_schema.dist_table" to "tenant_schema"